internal/config/config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"flag"
+	"os"
+	"time"
+)
+
+type UserIDKey string
+
+const (
+	DefaultRunADDR                     = "localhost:8081"
+	DefaultDatabaseURI                 = "host=localhost port=5432 user=myuser password=mypassword dbname=mydatabase sslmode=disable"
+	DefaultAccrualSystemADRR           = "http://localhost:8080"
+	UserIDKeyName            UserIDKey = "userId"
+)
+
+type Config struct {
+	RunADDR     string
+	AccrualADDR string
+	Database
+	OrderPoll
+	ShutdownTime time.Duration // Время, которое даем для корректного завершения сервиса.
+}
+
+type Database struct {
+	DSN string
+}
+
+type OrderPoll struct {
+	PollSleepTime time.Duration
+	PollWorkerNum int
+}
+
+func Load() Config {
+	runADDR := flag.String("a", DefaultRunADDR, "отвечает за адрес запуска HTTP-сервера")
```

Comment:
Это не аббревиатура:

```suggestion
	runAddr := flag.String("a", DefaultRunADDR, "отвечает за адрес запуска HTTP-сервера")
```